azure-pipelines.yml

```diff
@@ -53,6 +53,15 @@ jobs:
     fetchDepth: 25
   - bash: ./checker/bin-devel/test-cf-inference.sh
     displayName: test-cf-inference.sh
+- job: cf_inference_jdk11
+  pool:
+    vmImage: 'ubuntu-16.04'
+  container: mdernst/cf-ubuntu-jdk11:latest
+  steps:
+    - checkout: self
```

Comment:
Please make the indentation consistent with the rest of the file.

Also, whenever you make a pull request, please check the "permit edits by maintainers" checkbox.  I tried to fix this myself but was unable to.